addons/medical_feedback/XEH_preInit.sqf

```diff
@@ -26,4 +26,741 @@ PREP_RECOMPILE_END;
+    ["A3\sounds_f\characters\human-sfx\P02\Low_hit_2",3.1622777,1,80],
+    ["A3\sounds_f\characters\human-sfx\P02\Low_hit_3",3.1622777,1,80],
+    ["A3\sounds_f\characters\human-sfx\P02\Low_hit_4",3.1622777,1,80],
+    ["A3\sounds_f\characters\human-sfx\P02\Low_hit_5",3.1622777,1,80],
+    ["A3\sounds_f\characters\human-sfx\P02\Low_hit_6",3.1622777,1,80]
+], [
+    ["A3\sounds_f\characters\human-sfx\P02\Mid_hit_1",3.1622777,1,80],
+    ["A3\sounds_f\characters\human-sfx\P02\Mid_hit_2",3.1622777,1,80],
+    ["A3\sounds_f\characters\human-sfx\P02\Mid_hit_3",3.1622777,1,80],
+    ["A3\sounds_f\characters\human-sfx\P02\Mid_hit_4",3.1622777,1,80]
+], [
+    ["A3\sounds_f\characters\human-sfx\P02\Max_hit_1",3.1622777,1,80],
+    ["A3\sounds_f\characters\human-sfx\P02\Max_hit_2",3.1622777,1,80],
+    ["A3\sounds_f\characters\human-sfx\P02\Max_hit_3",3.1622777,1,80],
+    ["A3\sounds_f\characters\human-sfx\P02\Max_hit_4",3.1622777,1,80]
+]]];
+
+GVAR(HitScreamNamespace) setVariable ["Male09ENG", [[
+    ["A3\sounds_f\characters\human-sfx\P03\Hit_Low_1",3.1622777,1,80],
+    ["A3\sounds_f\characters\human-sfx\P03\Hit_Low_2",3.1622777,1,80],
+    ["A3\sounds_f\characters\human-sfx\P03\Hit_Low_3",3.1622777,1,80]
+], [
+    ["A3\sounds_f\characters\human-sfx\P03\Hit_Mid_1",3.1622777,1,80],
+    ["A3\sounds_f\characters\human-sfx\P03\Hit_Mid_2",3.1622777,1,80],
+    ["A3\sounds_f\characters\human-sfx\P03\Hit_Mid_3",3.1622777,1,80]
+], [
+    ["A3\sounds_f\characters\human-sfx\P03\Hit_Max_1",3.1622777,1,80],
+    ["A3\sounds_f\characters\human-sfx\P03\Hit_Max_2",3.1622777,1,80],
+    ["A3\sounds_f\characters\human-sfx\P03\Hit_Max_3",3.1622777,1,80]
+]]];
+
+GVAR(HitScreamNamespace) setVariable ["Male07ENG", [[
+    ["A3\sounds_f\characters\human-sfx\P04\Hit_Low_1",3.1622777,1,80],
+    ["A3\sounds_f\characters\human-sfx\P04\Hit_Low_2",3.1622777,1,80],
+    ["A3\sounds_f\characters\human-sfx\P04\Hit_Low_3",3.1622777,1,80],
+    ["A3\sounds_f\characters\human-sfx\P04\Hit_Low_4",3.1622777,1,80],
+    ["A3\sounds_f\characters\human-sfx\P04\Hit_Low_5",3.1622777,1,80]
+], [
+    ["A3\sounds_f\characters\human-sfx\P04\Hit_Mid_1",3.1622777,1,80],
+    ["A3\sounds_f\characters\human-sfx\P04\Hit_Mid_2",3.1622777,1,80],
+    ["A3\sounds_f\characters\human-sfx\P04\Hit_Mid_3",3.1622777,1,80],
+    ["A3\sounds_f\characters\human-sfx\P04\Hit_Mid_4",3.1622777,1,80],
+    ["A3\sounds_f\characters\human-sfx\P04\Hit_Mid_5",3.1622777,1,80]
+], [
+    ["A3\sounds_f\characters\human-sfx\P04\Hit_Max_1",3.1622777,1,80],
+    ["A3\sounds_f\characters\human-sfx\P04\Hit_Max_2",3.1622777,1,80],
+    ["A3\sounds_f\characters\human-sfx\P04\Hit_Max_3",3.1622777,1,80],
+    ["A3\sounds_f\characters\human-sfx\P04\Hit_Max_4",3.1622777,1,80],
+    ["A3\sounds_f\characters\human-sfx\P04\Hit_Max_5",3.1622777,1,80]
+]]];
+
+GVAR(HitScreamNamespace) setVariable ["Male03GRE", [[
+    ["A3\sounds_f\characters\human-sfx\P05\Hit_Low_1",3.1622777,1,80],
```

Comment:
I feel like we should be able to cut down on the amount of repeat code here with macros

Reply:
or can we not even pull this data from config somehow? It's done preInit so any extra overhead is imo worth reducing the mass of lines.

Reply:
We should actually even move it to preStart so it only gets done once ever! 👍